TW-1102 Postbot section (#4922)
postmanlabs/postman-docs: src/pages/docs/getting-started/about-postbot.md

---
title: "About Postbot"
updated: 2023-07-17
contextual_links:
  - type: section
    name: "Additional resources"
  - type: subtitle
    name: "Blog posts"
  - type: link
    name: "Generative AI and the impact on APIs and software development"
    url: "https://blog.postman.com/generative-ai-and-the-impact-on-apis-and-software-development/"
  - type: link
    name: "Introducing Postbot, Postman’s new AI assistant"
    url: "https://blog.postman.com/introducing-postbot-postmans-new-ai-assistant/"
---

> > **Postbot is in beta for [Free, Basic, and Professional plans.](https://www.postman.com/pricing/)**

Reviewer: ```suggestion
> **Postbot is in beta for [Free, Basic, and Professional plans](https://www.postman.com/pricing/).**
```

Author: None of these pricing blurbs have a period at the end. (Not sure if that's the right call, but I don't want to change all of them here.)

Reviewer: Sounds good. I just moved it out of the brackets so that it doesn’t get hyperlinked.

> Postbot is your AI assistant for API workflows in Postman. It can help you debug and understand APIs, write tests faster, write Postman Flows, make sense of large quantities of data, and more.

Reviewer: ```suggestion
Postbot is an AI assistant for your Postman API workspace. It can help you debug and understand APIs, write tests faster, write Postman Flows, and make sense of large quantities of data.
```

> To use Postbot, look for the Postbot icon <img alt="Postbot icon" src="https://assets.postman.com/postman-docs/v10/icon-postbot-v10-16.jpg#icon" width="18px">. Select it and enter a plan text prompt, and Postbot will help you complete your work.

Reviewer: ```suggestion
To use Postbot, select the Postbot icon <img alt="Postbot icon" src="https://assets.postman.com/postman-docs/v10/icon-postbot-v10-16.jpg#icon" width="18px">. As you begin typing, Postbot offers different prompts to help you complete your task.
```

> There are several places you can use Postbot:

Reviewer: ```suggestion
You can use Postbot to:
```

> * Writing tests - Postbot can add tests to your API requests, based on your prompt. Use Postbot to add a new set of tests, visualize responses, save a field from a response, or fix your existing tests. For more information, see [using Postbot to write tests](/docs/writing-scripts/test-scripts#using-postbot-to-write-tests).

Reviewer: ```suggestion
* Write tests - Postbot adds tests to your API requests based on your prompt. Add a new set of tests, visualize responses, save a field from a response, or fix your existing tests. For more information, see [using Postbot to write tests](/docs/writing-scripts/test-scripts#using-postbot-to-write-tests).
```

> * Writing FQL in Postman Flows - _Flows Query Language_ (FQL) can be used to parse and transform JSON data to get the fields and structure you want. You can ask Postbot to generate FQL for you. Enter an idea for a query in simple text, and Postbot will suggest an FQL query for your. For more information, see [Using Postbot with FQL](/docs/postman-flows/flows-query-language/introduction-to-fql/#using-postbot-with-fql)

Reviewer: ```suggestion
* Write FQL queries in Postman Flows - _Flows Query Language_ (FQL) queries can parse and transform JSON data to get the fields and structure you want. You can ask Postbot to generate FQL queries for you. Enter a search idea in natural language, and Postbot suggests an FQL query. For more information, see [Using Postbot with FQL](/docs/postman-flows/flows-query-language/introduction-to-fql/#using-postbot-with-fql).
```

> We are excited about the longer-term potential of AI and the valuable role it can play. Stay tuned for more new Postbot updates as we continue to add more new features to automate how you work with APIs.

Reviewer: Sounds a bit like marketing copy. If there are additional features in development, consider saying something like, “Postman is working on additional AI features. Sign up to stay informed about the updates.”
